src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java

```diff
@@ -0,0 +1,95 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.SurveillanceReportPage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+
+/**
+ * Class SurveillanceReportPageAsserts definition.
+ */
+public class SurveillanceReportPageAsserts extends Base {
+
+    /**
+     * Assert that CHPL Surveillance is the expected text.
+     * @param expectedPageTitle is CHPL Surveillance
+     */
+    @Then("^I see \"([^\"]*)\" as the page title for the surveillance report page$")
+    public void iSeePageTitle(final String expectedPageTitle) {
+        WebElement link = SurveillanceReportPage.surveillancePageSubtitle(getDriver());
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].scrollIntoView();", link);
+        String actualPageTitle = SurveillanceReportPage.surveillancePageTitle(getDriver()).getText();
+        assertEquals(expectedPageTitle, actualPageTitle);
+    }
+
+    /**
+     * Assert that ACBs are expected to be displayed under available reports.
+     * @param expectedACBs are Drummond Group, ICSA Labs, SLI Compliance & UL LLC
+     */
+    @Then("^I see available reports for \"([^\"]*)\"$")
+    public void iSeeReportsAvailableFor(final String expectedACBs) {
+        String actualACBs = SurveillanceReportPage.availableYearQuarterForAcbs(getDriver(), expectedACBs).getText();
+        String[] acbTempArray = actualACBs.split("\n");
+        StringBuffer buff = new StringBuffer();
+        for (String string : acbTempArray) {
```

Comment:
What is this for? You're taking a string like `Name1 \n Name 2 \n Name 3`, splitting it into an array, then joining the array again into a string like `Name1 Name 2 Name 3`, right? Then seeing if the "expectedACBs" string is in that string? What the split and join? It'd work with `\n`s in the middle of the string, right?

If anything, the scenario(s) should look for each ACB on its own, because that's what a user actually sees, not this odd "concatenated list of ACB names"